requirements/docs.txt

```diff
@@ -6,3 +6,4 @@ sphinx-gallery
 sphinx-copybutton
 pytest-runner
 scikit-learn
+matplotlib>=2.0.0
```

Comment:
Why dont we blacklist ,!=3.0.0 here as well?

Reply:
you are right, I just wanted to hear back more general concerns about the methodology before cleaning this kind of stuff up. In fact, we definitely need to blacklist 3.0.0 for the docs.